misc.php

```diff
@@ -79,3 +79,19 @@ function action_wpcom_vip_verify_string() {
 		}
 	}
 }, 10, 1 );
+
+/**
+ * Hooks pre_ping to stop any pinging from happening,
+ * unless `WPCOM_VIP_DO_PINGS` is set to `true`.
+ *
+ * @param array $post_links The URLs to be pinged
+ */
+function wpcom_vip_pre_ping( $post_links ) {
+	if ( ! defined( 'WPCOM_VIP_DO_PINGS' ) || true !== WPCOM_VIP_DO_PINGS ) {
+		return;
```

Comment:
Scratch what I said about returning void, I looked at the actual call to this action and it's not expecting a return value, :facepalm:.

The logic is still backwards - we want to return without modifying the `$post_links` if the constant _is_ true, not if it's not true.